From 8afc91270824f0e4aaa038830fa32a163a416b8b Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Wed, 9 Oct 2024 13:06:00 +0200
Subject: [PATCH] add win pthread

---
 src/Makevars.win | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makevars.win b/src/Makevars.win
index 437a261..6e1e04c 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -14,7 +14,7 @@ endif
 ifeq ($(PKG_CPPFLAGS),)
 RWINLIB = ../windows/protobuf
 PKG_CPPFLAGS = -I$(RWINLIB)/include -DUSENEWAPI
-PKG_LIBS = -L$(RWINLIB)/lib${subst gcc,,${COMPILED_BY}}${R_ARCH} -L$(RWINLIB)/lib -lprotobuf
+PKG_LIBS = -L$(RWINLIB)/lib${subst gcc,,${COMPILED_BY}}${R_ARCH} -L$(RWINLIB)/lib -lprotobuf -pthread
 PROTOC_DIR = $(RWINLIB)/bin$(subst 64,,$(WIN))/
 endif
 
